### DISCUSSION TITLE
DATABASE_HOST=database

### DISCUSSION THREAD

**Discussion by tony06077** [tobiasehlert/teslamateapi#98] [Q&A]
Dear friends
I have meet a problem that when i want to add api tp teslamate .THE "DATABASE_HOST=database" which i need to replace it ?
The error shows DATABASE_HOST which is undefined .Is it possible to highlight some areas  in the code that need to be changed ? Another question is whethere there are video Tutorial video. I using tasker and tesla plugin tasker to control tesla now .

Tony

**Comment by tobiasehlert** [comment]
Hi @tony06077,

Sorry for the slow reply!

Regarding the host parameter, that should be the same value as TeslaMate configuration in the same _docker-compose.yaml_ file.
Can you please share what environment variables you are setting when you start the docker container?
Please mask the variable in a sensible way, so you don't expose too much information of your setup.

Question regarding tutorial video, we do not have that :)

Regards,
Tobias
